src/main/java/seedu/hustler/game/shop/items/ShopItem.java

```diff
@@ -0,0 +1,82 @@
+package seedu.hustler.game.shop.items;
+
+/**
+ * The class in which every item will inherit from.
+ */
+public abstract class ShopItem implements Purchasable {
```

Comment:
These shopitems may be better programmed as data where by you read item details from some data file instead of managing these items as classes/objects. In the future where this application has hundreds of items, it would be very tough to modify the code base. For instance when adding some metric of an item. But since this is already done, good job.